django/db/models/expressions.py

```diff
@@ -202,13 +202,13 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
 
         Returns: an Expression to be added to the query.
         """
-        c = self.copy()
-        c.is_summary = summarize
-        c.set_source_expressions([
+        clone = self.copy()
```

Comment:
It would be better to make these renames in a separate commit since it's an unrelated cosmetic cleanup.